engine/modelc/src/modelimporter_gltf.cpp

```diff
@@ -899,6 +898,29 @@ static void LoadPrimitives(Scene* scene, Model* model, cgltf_data* gltf_data, cg
             uint32_t size = mesh->m_VertexCount * mesh->m_TexCoords0NumComponents;
             InitSize(mesh->m_TexCoords0, size, size);
         }
+
+        if (prim->indices)
+        {
+            ReadAccessorUint32ToArray(prim->indices, 1, mesh->m_Indices);
+        }
+        else
+        {
+            // for now, we only support triangles
+            if(prim->type == cgltf_primitive_type_triangles)
+            {
+                uint32_t num_vertices = mesh->m_Positions.Size() / 3;
+                mesh->m_Indices.SetCapacity(num_vertices);
+                mesh->m_Indices.SetSize(num_vertices);
+                for (uint32_t i = 0; i < num_vertices; ++i)
+                {
+                    mesh->m_Indices[i] = i;
+                }
+            }
+            else if (prim->type == cgltf_primitive_type_points)
+            {
+                dmLogWarning("Primitive in mesh %s uses point type which we don't support.", mesh->m_Name);
+            }
```

Comment:
Not sure if we'll support points anytime soon. The still don't feel very useful imho.